### PR TITLE
v3.1.0 Skippable support

### PR DESCRIPTION
**Pull request by jdcba**
`skippable { ??? }` is a replacement for (or wrapper around) specs2’s `{ ??? }.orSkip`, adding support for specs with `ThrownExpectations`.

(This will facilitate CI QA tests that generate warnings instead of errors.)

### CODE REVIEW

**Comment by jdcba** [comment]
@rowandavies / anyone?

**Comment by jdcba** [comment]
Everyone okay for me to merge this?

File: src/test/scala/au/com/cba/omnia/test/SkippableSpec.scala [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  }
+
+  def testCustomSkippedExceptionMessage = {
+    customExceptionalSpec.testFailure must throwA[SkipException].like {
+      case SkipException(skipped) => skipped.message must_==
+        s"Skipped: ${customExceptionalSpec.testFailureMessage}"
+    }
+  }
+
+  def testScaladocBehaviour = {
```
Comment (rowandavies) [inline src/test/scala/au/com/cba/omnia/test/SkippableSpec.scala]: I think I'm missing something here. What does this test, and how does it relate to scaladoc?

Comment (jdcba) [inline src/test/scala/au/com/cba/omnia/test/SkippableSpec.scala]: This tests (“Skippable should: match Scaladoc behaviour”) that the sample code in the Scaladoc is compilable and works as intended.

**Pull request merged**